### ISSUE TITLE
[Question]How to use early stopping in advisor when I train a CNN model?

### ISSUE THREAD

**Issue by githubche** [tobegit3hub/advisor#19]

**Comment by tobegit3hub** [comment]
Advisor is model-agnostic and you can get the recommended parameters and train easily. We have the examples of [tensorflow mnist](https://github.com/tobegit3hub/advisor/tree/master/advisor_client/examples/tensorflow_mnist) so that you can refer to them.

Now we have some early stopping algorithm but they require more data and not efficient.

**Comment by githubche** [comment]
I have seen the examples, they just output the final objective such as accuracy. I wonder how to output the training step information, and let advisor help to early stop the trial, avoiding over fitting.

**Comment by ed-word** [comment]
@githubche did you manage to find a solution?
